src/contexts/Staking/index.tsx

```diff
@@ -201,52 +204,39 @@ export const StakingProvider = ({
     if (api !== null && isReady && metrics.activeEra.index !== 0) {
       const previousEra = metrics.activeEra.index - 1;
 
-      // subscribe to staking metrics
-      const unsub = await api.queryMulti<AnyApi>(
+      const u = await api.queryMulti<AnyApi>(
         [
           api.query.staking.counterForNominators,
           api.query.staking.counterForValidators,
           api.query.staking.maxNominatorsCount,
           api.query.staking.maxValidatorsCount,
           api.query.staking.validatorCount,
-          [api.query.staking.erasValidatorReward, previousEra],
-          [api.query.staking.erasTotalStake, previousEra],
+          [api.query.staking.erasValidatorReward, previousEra.toString()],
+          [api.query.staking.erasTotalStake, previousEra.toString()],
           api.query.staking.minNominatorBond,
-          api.query.staking.historyDepth,
           [api.query.staking.payee, activeAccount],
+          [
+            api.query.staking.erasTotalStake,
+            metrics.activeEra.index.toString(),
+          ],
         ],
-        ([
-          _totalNominators,
-          _totalValidators,
-          _maxNominatorsCount,
-          _maxValidatorsCount,
-          _validatorCount,
-          _lastReward,
-          _lastTotalStake,
-          _minNominatorBond,
-          _historyDepth,
-          _payee,
-        ]) => {
+        (q) => {
```

Comment:
It's not typical for JS and TS to use such sort names. Could you please give `q` and `u` more meaningful names?

Reply:
Code copied from polkadot parent repo. I can change though if necessary